mega-zep-backend/src/main/resources/application.yaml

```diff
@@ -52,7 +52,8 @@ mega:
   mega:
     dash-url: http://localhost:4444/index.html
     zep:
-      url: https://www.zep-online.de/zepgepardecservices_test/sync/soap.php?v=7
+      url: https://www.zep-online.de/zepgepardecservices_test
```

Comment:
@Ruhsi   
Rename 'url'          -> 'origin'         because it is now just a origin
Rename 'soap-url' -> 'soap-path' because it is a path not a url

Resulting parameters are 'zep.origin' and 'zep.soap-path'
This is just for better naming